src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__link {
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav {
+    display: flex;
+    align-items: center;
+    height: 60px;
+
+    &-list {
+      display: flex;
+      list-style: none;
+      padding: 0;
+      height: 100%;
+    }
+
+    &-item {
+      position: relative;
+      height: 100%;
+      margin: 0 10px;
+    }
+
+    &-link {
+      display: flex;
+      align-items: center;
+      text-decoration: none;
+      height: 100%;
+      font-size: $font-size-medium;
+      color: $secondary-color;
+      text-transform: uppercase;
+      transition-duration: 300ms;
+      font-weight: $font-weight-medium;
+
+      &:hover {
+        color: $primary-color;
```

Comment:
Check if the `$primary-color` variable is defined in 'src/styles/utils/variables.scss' and follows the Code Style Rules for color palette management.